test/unit/streaming.models.MediaPlayerModel.js

```diff
@@ -302,6 +302,22 @@ describe('MediaPlayerModel', function () {
         expect(stableBufferTime).to.equal(10);
     });
 
+    it('should configure HybridSwitchBufferTime', function() {
```

Comment:
Minor, but could this include the stableBufferTime too, just to ensure that the OR guard is the right way round if both exist?

Reply:
Good shout!

Reply:
Done